src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -497,4 +503,18 @@ int findHighestCaseNumberWhenCaseNumbersExist(List<TestCase> testCases) {
         .get(0)
         .getCaseNumber();
   }
+
+  void convertDateTimeToUTC(TestCase testCase) {
```

Comment:
You are mutating the input testCase object directly, Although it is ok to do so, to avoid any misuse of this method, or any unexpected calls,
1. I think we should make this method private
2. Do not mutate the object, instead take json as input and return updated json.
Either way its good

Reply:
@RohitKandimalla ok done.